doc/md/guides/quickstart.mdx

```diff
@@ -10,11 +10,31 @@ import Admonition from '@theme/Admonition';
 
 # Quickstart
 
-Welcome to the Holos Quickstart guide.  Holos is an open source tool to manage
-software development platforms safely, easily, and consistently.  We'll use
-Holos to manage a fictional bank's platform, the Bank of Holos.  In doing so
-we'll take the time to explain the foundational concepts of Holos.
-
+Welcome to the Holos Quickstart guide.  Holos is an open source tool that helps
+you build and maintain your software development platform safely and quickly.
+
+Before Holos we used Helm, Kustomize, and bespoke scripts to glue together all
+of the software that comprises a development platform.  This worked briefly,
+but scaled poorly because:
+
+  - We lacked a centralized way to track the configuration data needed to
+  model and deploy a software component.
+  - There was no native way to validate data values, which meant that unsafe changes that didn't cause build failures might be missed and deployed.
+  - Updating each software component was very painful. Components frequently used different versions of our implementation code, which created confusion and a maintenance burden.
+  - Updating each software component was very painful, and needed to be done every time our
+  implementation code changed.
+
+In searching for solutions to these problems, we found that many organizations
+were creating the same kinds of implementation "glue code." There was a gap for
+standardized tooling to allow you to express the data needed to deploy these
+software components in a safe, cogent, and auditable way. Holos became the tool
+we created to solve these problems.
+
+In this guide we will walk you through the configuration required to deploy one of
+the components that make up a fictional software development platform, the Bank
+of Holos.  In doing so we'll take the time to explain the foundational concepts
+of Holos.
```

Comment:
The guides should use "we" most of the time and avoid "you" to establish a consistent voice.